lib/install.js

```diff
@@ -520,7 +520,8 @@ Installer.prototype.executeActions = function (cb) {
     [lock, node_modules, '.staging'],
     [rimraf, staging],
     [doParallelActions, 'extract', staging, todo, cg.newGroup('extract', 100)],
-    [doReverseSerialActions, 'remove', staging, todo, cg.newGroup('remove')],
+    [doReverseSerialActions, 'unbuild', staging, todo, cg.newGroup('unbuild')],
+    [doSerialActions, 'remove', staging, todo, cg.newGroup('remove')],
```

Comment:
This change will remove all exists packages on `node_modules` before install action start.